ui/assets/js/common.js

```diff
@@ -13,7 +13,8 @@ var lib = {
     UrlToRepo: function(repo, path, line, rev) {
         var url = repo.url.replace(/\.git$/, ''),
             pattern = repo['url-pattern'],
-            anchor = line ? lib.ExpandVars(pattern.anchor, { line : line }) : '';
+            filename = path.replace(/^.*\//, ''),
```

Comment:
Oh I see, this is extracting the basename. Seems like there is a more straightforward approach with `lastIndexOf`. `path.substring(path.lastIndexOf('/')+1)`?